packages/appChrome/components/HeaderBar.tsx

```diff
@@ -30,19 +29,22 @@ class Header extends React.PureComponent<HeaderProps, {}> {
         );
         return css`
           background-color: ${bgColor};
-          padding-left: ${spaceSizes[props.theme.headerPaddingHor] ||
```

Comment:
`spaceSizes.none` is `0`, which is falsy. This caused the it to fall back to the value after the `||` operator

Reply:
All the changes are the same, just with different theme property names